tmux.1

```diff
@@ -1803,6 +1804,7 @@ The following commands are supported in copy mode:
 .It Li "pipe-and-cancel [<command>] [<prefix>]" Ta "" Ta ""
 .It Li "previous-matching-bracket" Ta "" Ta "M-C-b"
 .It Li "previous-paragraph" Ta "{" Ta "M-{"
+.It Li "previous-prompt" Ta "" Ta ""
```

Comment:
> Please update tmux.1 with the new commands.

Added these.

But do you think these commands need some additional documentation? Because unless the shell is sending OSC 133, these command won't actually work. Do we need to clarify that in the manual?